### DISCUSSION TITLE
Can my company use Polaris. I am unsure reading the License?

### DISCUSSION THREAD

**Discussion by mccxiv** [Shopify/polaris-react#8607] [Q&A] [answered]
Q: Can my company use this repository to develop software that does not integrate or interoperate with Shopify software or services?

I'm not a lawyer but the wording on the license can be interpreted (by me) in any number of ways that contradict each other...

**Comment by alex-page** [comment, marked as answer]
@mccxiv I am not a lawyer and I don't think Shopify's lawyers will jump into GitHub to discuss this unfortunately. I also don't know what your company is or what the product that you are making. Even if I did I don't think I could give you a great answer.

During Covid-19 I saw Polaris being used to help Government sectors build critical software to help citizens find if they had been exposed and get notifications if they need to be vaccinated. This was celebrated and a good example of where Polaris is allowed to be used.

I would read our license and make the best judgement call you can: https://github.com/Shopify/polaris/blob/main/LICENSE.md
